plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/write/MapKeyValuesSchemaConverter.java

```diff
@@ -172,41 +171,42 @@ public static GroupType mapType(Repetition repetition, String alias, String mapA
     {
         //support projection only on key of a map
         if (valueType == null) {
-            return listWrapper(
-                    repetition,
+            return mapKvWrapper(
```

Comment:
Let's not mix refactoring of tests with removal of deprecated API usage.
Please keep the changes restricted only to removal of deprecated API usage here, the refactoring can be done separately.

Reply:
Reverted these changes. Will split into a separate PR. Thanks.

Reply:
Created #13837 for this refactor.